ProcessOut/Classes/ProcessOut.swift

```diff
@@ -365,6 +366,34 @@ public class ProcessOut {
         }
     }
     
+    /// Increments the authorization of an applicable invoice by a given amount
+    ///
+    /// - Parameters:
+    ///   - invoiceId: Invoice generated on your backend
+    ///   - amount: The amount by which the authorization should be incremented
+    ///   - handler: Custom 3DS2 handler (please refer to our documentation for this)
+    public static func incrementAuthorizationAmount(invoiceId: String, amount: Int, handler: ThreeDSHandler) {
+        let incrementRequest = IncrementAuthorizationRequest(amount: amount)
+        guard let body = try? JSONEncoder().encode(incrementRequest) else {
+            handler.onError(error: ProcessOutException.InternalError)
+            return
+        }
+        
+        do {
+            let json = try JSONSerialization.jsonObject(with: body, options: []) as! [String : Any]
+            HttpRequest(route: "/invoices/" + invoiceId + "/increment_authorization", method: .post, parameters: json, completion: {(data, error) -> Void in
```

Comment:
> Improvement : You can avoid passing json here inside the HttpRequest.

Thank you for the improvement suggestion regarding the way data is passed to `HttpRequest`. As a future improvement this would be beneficial, but I think that it is outside the scope of this PR. In the meantime, I have updated the `json` object assignment statements to eliminate the forced down castings like you suggested.

> naming `HttpRequest` -> `httpRequest` as how other methods are named

Method naming consistency is an issue throughout this class and there are several public methods where the capitalisation is inconsistent. Since it would be a backwards-incompatible API change to fix all of these I don't think it is within the scope of this PR. I would like to address this at a future time.